styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
+  padding-top: 50px;
+  padding-bottom: 50px;
```

Comment:
ok, got it.
For the last section, when it clicks to the target, it does not scroll all the way to the top. probs because the page has ended. how can i resolve that?